packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -481,9 +481,20 @@ export class SpecBuilder {
     return this;
   }
 
-  watchSnapshots(version: TestSnapshotVersion): SpecBuilder {
+  watchSnapshots(
+    version: TestSnapshotVersion,
+    targets?: Query[],
+    resumeToken?: string
+  ): SpecBuilder {
     this.assertStep('Watch snapshot requires previous watch step');
-    this.currentStep!.watchSnapshot = version;
+
+    if (this.currentStep!.watchSnapshot !== undefined) {
```

Comment:
Could we just always make the watchSnapshot a separate step (i.e. remove this check? and perhaps the assert above?)

Reply:
Yeah, I considered that (and added it as a suggestion to the PR description. cough cough). Now consider it done.